src/databricks/labs/ucx/hive_metastore/mapping.py

```diff
@@ -58,6 +58,10 @@ def from_src_dst(cls, src_table: TableInfo, dst_schema: SchemaInfo) -> "Rule":
     def match(self, table: TableIdentifier) -> bool:
         return table.catalog == "hive_metastore" and self.src_schema == table.schema and self.src_table == table.table
 
+    @property
+    def as_uc_table(self) -> Table:
+        return Table(self.catalog_name, self.dst_schema, self.dst_table)
+
     @property
     def as_uc_table_key(self) -> str:
```

Comment:
Not for this PR, but would try to avoid using this  key and use the `Table` data class instead